feat: User awaiting approval (#680)
Substra/substra-backend: charts/substra-backend/README.md

| `oidc.users.useRefreshToken`            | Attempt to refresh user info in the background.                                                                                                                 | `true`  |
| `oidc.users.loginValidityDuration`      | How long a user account is valid after an OIDC login, in seconds                                                                                                | `3600`  |
| `oidc.users.channel`                    | The channel to assign OIDC users to (mandatory)                                                                                                                 | `nil`   |

> | `oidc.users.requireApproval`            | This switches on the user approval. A user using the OIDC login for the first time will need approval from an admin. Is not compatible with default channel     | `false` |

Reviewer: nitpicking:
```suggestion
| `oidc.users.requireApproval`            | Activate the user approval. A user using the OIDC login for the first time will need approval from an admin. It is not compatible with default channel     | `false` |
```
